pkg/authorization/api/types.go

```diff
@@ -174,6 +174,11 @@ type ResourceAccessReviewResponse struct {
 	Users sets.String
 	// Groups is the list of groups who can perform the action
 	Groups sets.String
+
+	// EvaluationError is an indication that some error occurred during resolution, but partial results can still be returned.
+	// It is entirely possible to get an error and be able to continue determine authorization status in spite of it.  This is
+	// most common when a bound role is missing, but enough roles are still present and bound to reason about the request.
+	EvaluationError string
```

Comment:
ha, you added it to SubjectRulesReviewStatus already... now you are using my desire for symmetry against me